user-guide/en-US/All.asciidoc

```diff
@@ -800,60 +800,122 @@ Just as with the Authorization policy, you can define any number of rules you'd
 ==== IP Whitelist Policy
 [[policy-ip-whitelist]]
 ===== Description
-DESCRIPTION TBD HERE
+The IP Whitelist Policy Type is the counterpart to the IP Blacklist Policy type. In the IP Whitelist policy, only inbound API requests from application clients that satisfy the policy are accepted. 
```

Comment:
> requests from application clients that

May want to change that phrasing slightly as it can be on ClientApp, Policy, or API. I know you're trying to say "the thing calling apiman", but I'm not sure we've settled on a standard term for that (have we?)